docs/api/field_types_reference/richtextfield.md

```diff
@@ -8,29 +8,38 @@ This Field Type validates and stores structured rich text, and exposes it in sev
 |------|------|------|
 |`RichText`|`ezrichtext`|mixed|
 
-## PHP API Field Type 
+## PHP API Field Type 
 
-### Input expectations
+### Value object
+
+`EzSystems\EzPlatformRichText\eZ\FieldType\RichText\Value` offers the following properties:
 
-|Type|Description|Example|
+|Property|Type|Description|
 |------|------|------|
-|`string`|XML document in one of the Field Type's input formats as a string.|See the example below.|
-|`DOMDocument`|XML document in one of the Field Type's input formats as a `DOMDocument` object.|See the example below.|
-|`EzSystems\EzPlatformRichText\eZ\FieldType\RichText\Value`|An instance of the Field Type's `Value` object.|See the example below.|
+|`xml`|`DOMDocument`|Internal format value as an instance of `DOMDocument`.|
+
+### Input expectations
+
+|Type|Description|
+|------|------|
+|`string`|XML document in one of the Field Type's input formats as a string.|
+|`DOMDocument`|XML document in one of the Field Type's input formats as a `DOMDocument` object.|
+|`EzSystems\EzPlatformRichText\eZ\FieldType\RichText\Value`|An instance of the Field Type's `Value` object.|
 
 ##### Input formats
 
-The Field Type works with XML and also expects an XML value as input, whether as a string, `DOMDocument` object or Field Type's `Value` object. When the value is given as a string or a `DOMDocument` object, it will be checked for conformance with one of the supported input formats, then dispatched to the appropriate converter, to be converted to the Field Type's internal format. No conversion will be performed if providing the value in Field Type's internal format or as Field Type's `Value` object. In the latter case it will be expected that the `Value` object holds the value in the Field Type's internal format.
+The Field Type expects an XML value as input, in the form of a string, `DOMDocument` object, or Field Type's `Value` object.
+The Field Type's `Value` object must hold the value in the Field Type's [internal format](#internal-format).
+For a string of a `DOMDocument` object, if the input des not conform to this format, it is converted into it.
 
-Currently supported input formats are described in the table below:
+!!! note "Legacy eZXML format"
 
-|Name|Description|
-|------|------|
-|[[= product_name =]]'s DocBook variant|Field Type's internal format.|
-|XHTML5 editing format|Typically used with in-browser HTML editor.|
-|Legacy eZXML format|Compatibility with legacy XML format, used by [XmlText Field Type](xmltextfield.md).|
+    The Legacy eZXML format is also supported by the RichText Field Type,
+    but can be used only for migration, in compatibility with the [legacy XML format](xmltextfield.md).
```

Comment:
Maybe we can emphasize what tools should be used to perform the migration, ref: https://github.com/ezsystems/ezplatform-xmltext-fieldtype#migrating-from-xmltext-to-richtext (if this has not been mentioned already)?

Reply:
@konradoboza I guess we could link to https://doc.ibexa.co/en/master/migrating/migrating_from_ez_publish_platform/#321-migrate-xmltext-content-to-richtext instead?

Reply:
Sure, let's go with that reference.